EVGEN/AliGenPerformance.cxx

```diff
@@ -88,35 +90,35 @@ AliGenPerformance::AliGenPerformance():
 }
 //-----------------------------------------------------------------------------
 AliGenPerformance::AliGenPerformance(const char* generName, Int_t verboseLevel):
-  AliGenerator(),
-  fNJets(1),                // mean number of jets per event
-  fF1Momentum(0),           // momentum distribution function 
-  fFPhi(0),                 // phi distribution function
-  fFTheta(0),               // theta distribution function
-  fFPosition(0),            // position distribution function 
-  fFPdg(0),                 // pdg distribution function  
-  fStreamer(0),           // test stream - used for tuning of parameters of generator
-  fVerboseLevel(0)            // verbose level
+        AliGenerator(),
+        fNJets(1),                // mean number of jets per event
+        fF1Momentum(0),           // momentum distribution function
+        fFPhi(0),                 // phi distribution function
+        fFTheta(0),               // theta distribution function
+        fFPosition(0),            // position distribution function
+        fFPdg(0),                 // pdg distribution function
+        fStreamer(0),           // test stream - used for tuning of parameters of generator
+        fVerboseLevel(0)            // verbose level
```

Comment:
I know that I can see them with ?w=1. But I believe these changes are not necessary and only cause confusion with `git blame` etc. I would like to express again my opinion that we should try to avoid reformatting changes altogether and definitely not mix them with real changes.